service/src/main/resources/application.yml

```diff
@@ -55,6 +55,19 @@ cbas:
     # How often to check downstream services for health:
     healthCheckIntervalSeconds: 300
 
+management:
+  endpoints:
+    web:
+      exposure:
+        include: info,health
+      base-path: /actuator
```

Comment:
Note: these APIs come with actuator but I don't (yet) want to replace our existing health monitoring, so these are being sent to `/actuator` paths, for now

Reply:
I don't really understand this comment but I'm curious to know more... I'll ask you about it later today